### PR TITLE
allow children in AppStory paragraphs

### PR DESCRIPTION
**Pull request by KrystofVavra**

### CODE REVIEW

File: src/components/AppStory/AppStoryBody/content/components/AppStoryParagraph/index.js [hunk 1]
```diff
@@ -3,15 +3,21 @@ import PropTypes from 'prop-types';
...
 
-const AppStoryParagraph = ({className, text}) => {
+const AppStoryParagraph = ({className, text, children}) => {
 	const classes = classnames('ptr-AppStoryParagraph', {}, className);
 
-	return <p className={classes}>{text}</p>;
+	return (
+		<p className={classes}>
+			{text}
+			{children}
```
Comment (vdubr) [inline src/components/AppStory/AppStoryBody/content/components/AppStoryParagraph/index.js:12]:
```suggestion
			{children ? children : null}
```

File: src/components/AppStory/AppStoryBody/content/components/AppStoryParagraph/index.js [hunk 2]
```diff
@@ -3,15 +3,21 @@ import PropTypes from 'prop-types';
...
+		</p>
+	);
 };
 
 AppStoryParagraph.propTypes = {
 	className: PropTypes.string,
 	text: PropTypes.string,
+	children: PropTypes.any,
```
Comment (vdubr) [inline src/components/AppStory/AppStoryBody/content/components/AppStoryParagraph/index.js:20]:
```suggestion
	children: PropTypes.node,
```

File: src/components/AppStory/AppStoryBody/content/components/AppStoryParagraph/index.js (hunk 1 above)
Comment (KrystofVavra) [reply src/components/AppStory/AppStoryBody/content/components/AppStoryParagraph/index.js]: resolved [3a1197d](https://github.com/gisat-panther/visat-components/pull/22/commits/3a1197de9198358e12230a534dc0ec80edc73593)

File: src/components/AppStory/AppStoryBody/content/components/AppStoryParagraph/index.js (hunk 2 above)
Comment (KrystofVavra) [reply src/components/AppStory/AppStoryBody/content/components/AppStoryParagraph/index.js]: resolved [3a1197d](https://github.com/gisat-panther/visat-components/pull/22/commits/3a1197de9198358e12230a534dc0ec80edc73593)

File: src/components/AppStory/AppStoryBody/content/components/AppStoryParagraph/index.js [hunk 3]
```diff
@@ -3,15 +3,16 @@ import PropTypes from 'prop-types';
...
+const AppStoryParagraph = ({className, children}) => {
 	const classes = classnames('ptr-AppStoryParagraph', {}, className);
 
-	return <p className={classes}>{text}</p>;
+	return <p className={classes}>{children ? children : null}</p>;
 };
 
 AppStoryParagraph.propTypes = {
 	className: PropTypes.string,
 	text: PropTypes.string,
```
[the remark this answers is not in the document]
Comment (KrystofVavra) [reply src/components/AppStory/AppStoryBody/content/components/AppStoryParagraph/index.js]: removed [6527c3c](https://github.com/gisat-panther/visat-components/pull/22/commits/6527c3cad663a5a278e1322b6b0806ffab1bb701)

File: src/components/AppStory/AppStoryBody/content/components/AppStoryHeadline/index.js [hunk 4]
```diff
@@ -3,15 +3,16 @@ import PropTypes from 'prop-types';
...
+const AppStoryHeadline = ({className, children}) => {
 	const classes = classnames('ptr-AppStoryHeadline', {}, className);
 
-	return <h1 className={classes}>{headline}</h1>;
+	return <h1 className={classes}>{children ? children : null}</h1>;
 };
 
 AppStoryHeadline.propTypes = {
 	className: PropTypes.string,
 	headline: PropTypes.string,
```
[the remark this answers is not in the document]
Comment (KrystofVavra) [reply src/components/AppStory/AppStoryBody/content/components/AppStoryHeadline/index.js]: removed [6527c3c](https://github.com/gisat-panther/visat-components/pull/22/commits/6527c3cad663a5a278e1322b6b0806ffab1bb701)

**Review by vdubr — APPROVED** [review]

**Pull request merged**